test/unit/log-specs.js

```diff
@@ -5,46 +5,76 @@ import sinon from 'sinon';
 import * as Logs from '../../lib/device-log/ios-simulator-log';
 import * as CrashLogs from '../../lib/device-log/ios-crash-log';
 import log from '../../lib/commands/log';
+import * as TeenProcess from 'teen_process';
 
 
 chai.should();
 chai.use(chaiAsPromised);
 
-describe('XCUITestDriver - startLogCapture', function () {
-  let startCaptureSpy, crashLogStub, iosLogStub;
-
-  before(function () {
-    const spy = {
-      startCapture: _.noop,
-    };
-    startCaptureSpy = sinon.spy(spy, 'startCapture');
-    crashLogStub = sinon.stub(CrashLogs, 'IOSCrashLog').callsFake(function () {
-      this.startCapture = _.noop;
+describe('logs', function () {
+  describe('startLogCapture', function () {
```

Comment:
The tests in `XCUITestDriver -startLog Capture` where moved into this nested 'describe' block. The contents of the test didn't change. The diff looks more complicated than it actually.